suite-native/module-passphrase/src/components/PassphraseScreen.tsx

```diff
@@ -0,0 +1,21 @@
+import { Screen } from '@suite-native/navigation';
+import { useNativeStyles } from '@trezor/styles';
+
+import { PassphraseScreenHeader } from './PassphraseScreenHeader';
+
+type PassphraseScreenProps = {
+    children: React.ReactNode;
+};
+
+export const PassphraseScreen = ({ children }: PassphraseScreenProps) => {
```

Comment:
nit: What about something like this?

```suggestion
export const PassphraseScreenWrapper = ({ children }: PassphraseScreenProps) => {
```

Reply:
Fixed

Reply:
filename was not changed.

Reply:
Could you also rename the other component for consistency?

eg. `PassphraseContentScreen` --> `TitledPassphraseScreenWrapper`